smoke/LeafDevice/LeafDevice.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <TargetFramework>netcoreapp2.1</TargetFramework>
-    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>False</TreatWarningsAsErrors>
```

Comment:
Why change this flag? What warnings shown up as errors during build? can we resolve those warnings?

Reply:
The warnings were the FxCop style warnings which were part of the SDK.

Reply:
CA2007 is the warning that seems to be coming from the FxCop settings in the SDK.

Reply:
Remember to set back to True when merging back to master.

Reply:
As noted in the description: Warnings as errors was disabled for the LeafDevice project on account of an issue with the preview SDK. Once the SDK is patched and released, this flag can be re enabled.